kettle/BUILD

```diff
@@ -41,6 +41,9 @@ py_test(
     data = ["//:buckets"],
     # idem
     local = True,
+    deps = [
```

Comment:
If you have only one import then `["foo"]` is valid, otherwise you need to follow this format.